src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public abstract class Task {
+    private  String taskDescription;
+    private int taskId;
+    private boolean isDone;
+
+    Task(String taskDescription,int taskId){
+        this.taskDescription = taskDescription;
+        this.taskId = taskId + 1;
+        this.isDone = false;
+    }
+
+    Task(){
+        this("",0);
+    }
+
+    public String displayStatusSymbol(){
+        return (isDone ? "\u2713" : "\u2718");
```

Comment:
Again, can use constants to make it easy for the reader to understand what is happening here.